plaidml/core/internal.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Intel Corporation.
+// Copyright 2019 Intel Corporation
```

Comment:
minor nit: should this date be modified to 2020 (or even 2019-2020)?

Reply:
I have no idea what our policy is for source file headings... I've been updating the date for new files only thus far.

Reply:
Fair enough. I just saw it updated in some places and not others, so I was just wondering
<hr>

As a sidenote, I have seen other groups within Intel just do a blanket update to 2020. Not saying that needs to be addressed right now or even anytime soon.